.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: scala
+scala:
+- 2.11.7
+jdk:
+- oraclejdk8
+sudo: false
+cache:
+  directories:
+  - $HOME/.sbt
+  - $HOME/.ivy2
+before_cache:
+  - find $HOME/.sbt -name "*.lock" -type f -delete -print
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete -print
+before_install:
+  - export TZ=Europe/London
```

Comment:
`-export TZ=Europe/London` is needed by Travis because otherwise [CachedTest](https://github.com/guardian/membership-frontend/blob/master/frontend/test/controllers/common/CachedTest.scala) unit test fails on line 22: 
```
"Convert to http date string" in {
    val theDate = new DateTime(2001, 5, 20, 12, 3, 4, 555)
    Cached.toHttpDateTimeString(theDate) mustEqual "Sun, 20 May 2001 11:03:04 GMT"
  }
```

The failure might be due to TeamCity and Travis being set to different timezones.